src/server/game/AI/SmartScripts/SmartScriptMgr.h

```diff
@@ -558,8 +558,19 @@ enum SMART_ACTION
     SMART_ACTION_RANDOM_SOUND                       = 115,    // soundId1, soundId2, soundId3, soundId4, soundId5, onlySelf
     SMART_ACTION_SET_CORPSE_DELAY                   = 116,    // timer
     SMART_ACTION_DISABLE_EVADE                      = 117,    // 0/1 (1 = disabled, 0 = enabled)
-
-    SMART_ACTION_END                                = 118
+    SMART_ACTION_SET_CAN_FLY                        = 119,    // 0/1 (3.3.5 only, NYI)
```

Comment:
Yup, already saw it in another PR, that's why i set placeholder with comments (3.3.5 only, NYI) :)

Maybe a bad idea, will remove it if required